docs/source/14_contribution/technical_steering_committee.md

```diff
@@ -0,0 +1,59 @@
+# Join the Technical Steering Committee
+
+The term "Technical Steering Committee (TSC)" describes the group of people that maintain Kedro. The TSC has many responsibilities surrounding the project's future development; you can read about our duties in our [Technical Charter](https://github.com/kedro-org/kedro/blob/main/kedro_technical_charter.pdf). We accept new members on the TSC as this fuels Kedro's continued development. You can see [a list of Kedro's current and past maintainers](../12_faq/01_faq.md#who-maintains-kedro).
+
+In this section, we detail:
+- [Responsibilities of a maintainer](#responsibilities-of-a-maintainer)
+- [Requirements to become a maintainer](#requirements-to-become-a-maintainer)
+- [Application process](#application-process)
+- [Voting process](#voting-process)
+
+## Responsibilities of a maintainer
+
+### Product development
+
+ - Be available for at least one full day per week to help with product development
+ - Attend community meetings to discuss the project plans and roadmap
+ - Project maintenance including security, updates, CI/CD, builds and infrastructure
+ - Prioritize the work following the product roadmap to move the project forward
+
+### Community management
+
+- Make sure that ongoing pull requests are moving forward at the right pace or closing them
+- Guide the community to use the right channel:
+  - [Github](https://github.com/kedro-org/kedro/) for feature requests and bug reports
+  - [GitHub discussions](https://github.com/kedro-org/kedro/discussions)
+  - [Discord](https://discord.gg/akJDeVaxnB)
+
+## Requirements to become a maintainer
+
+Just contributing does not make you a maintainer; you need to demonstrate commitment to Kedro's long-term success by
+working with existing maintainers for a period.
+
+We look for commitment markers with the following:
+
+- Being able to write solid code and collaborate with the team and community
+- Understanding the project's code base and internals
+- Doing pull requests from our backlog or roadmap; maintainers need to work towards a common goal
+- Learning how the team works, including processes for testing, quality standards and code review
+- Evidence of already having started pull requests and code reviews under the guidance of maintainers; including asking
+  for help where needed
+
+## Application process
+
+Quarterly, existing maintainers will curate a list of contributors that have shown regular activity on the project over the prior months. From this list, maintainer candidates are selected and proposed for a vote.
+
+Following a successful vote, candidates are added to the `kedro-developers` team on the Kedro GitHub organisation
+and the `kedro-team` group on the Kedro Discord organisation.
```

Comment:
Probably also after the candidates themselves have accepted to be maintainers. 😂